src/MAUI/Maui.Samples/Samples/UtilityNetwork/ConfigureSubnetworkTrace/ConfigureSubnetworkTrace.xaml

```diff
@@ -1,74 +1,98 @@
-                                   Text="" />
-                        </ScrollView>
-                    </ViewCell>
-                </TableSection>
-                <TableSection>
-                    <ViewCell>
-                        <Button Clicked="OnTrace" Text="Trace" />
-                    </ViewCell>
-                    <ViewCell>
-                        <Button Clicked="OnReset" Text="Reset" />
-                    </ViewCell>
-                </TableSection>
-            </TableRoot>
-        </TableView>
-    </Grid>
+﻿<ContentPage
+    x:Class="ArcGISRuntime.Samples.ConfigureSubnetworkTrace.ConfigureSubnetworkTrace"
+    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml">
+    <ScrollView MinimumHeightRequest="1000">
+        <Grid Style="{DynamicResource EsriSampleContainer}">
+            <StackLayout Padding="10" Spacing="10">
+                <Label FontSize="Title" Text="Trace options" />
+                <Grid ColumnSpacing="10" RowSpacing="10">
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="auto" />
+                        <ColumnDefinition Width="auto" />
+                    </Grid.ColumnDefinitions>
+                    <Grid.RowDefinitions>
+                        <RowDefinition />
+                        <RowDefinition />
+                    </Grid.RowDefinitions>
+                    <Label
+                        Grid.Row="0"
+                        Grid.Column="0"
+                        Text="Include barriers"
+                        VerticalTextAlignment="Center" />
+                    <Switch
+                        x:Name="IncludeBarriers"
+                        Grid.Row="0"
+                        Grid.Column="1"
+                        IsToggled="True"
+                        Toggled="IncludeBarriersChanged" />
+                    <Label
+                        Grid.Row="1"
+                        Grid.Column="0"
+                        Text="Include containers"
+                        VerticalTextAlignment="Center" />
+                    <Switch
+                        x:Name="IncludeContainers"
+                        Grid.Row="1"
+                        Grid.Column="1"
+                        IsToggled="True"
+                        Toggled="IncludeContainersChanged" />
+                </Grid>
+                <Label FontSize="Title" Text="Example barrier condition for this data: 'Transformer Load' Equal '15'" />
+                <Label Text="Define new condition" />
+                <Grid ColumnSpacing="10">
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="auto" />
+                        <ColumnDefinition Width="auto" />
+                        <ColumnDefinition Width="auto" />
+                        <ColumnDefinition Width="*" />
+                    </Grid.ColumnDefinitions>
+                    <Picker
```

Comment:
I'd stack these pickers vertically and give them a minimum width as they render very small on android. The entry is hard to see if the first two pickers have values.